internal/dao/sqlx/account_dao.go

```diff
@@ -14,6 +15,7 @@ import (
 )
 
 const (
+	createAccount             = `INSERT INTO accounts (account_number, org_id) VALUES ($1, $2) ON CONFLICT DO UPDATE RETURNING id`
```

Comment:
@ lzap [asked me](https://github.com/RHEnVision/provisioning-backend/pull/84#discussion_r920834394) to drop this, but I am not sure what was the idea behind the 'DO UPDATE' in the ticket.

Reply:
Oh yeah, that is basically wrong, I see, I'll look into that, I guess we can `DO NOTHING` for now :thinking: but erroring out and letting UI/User try again is the best way I guess.

Reply:
I just left it out, so we error out and let the User try again.